test/upgrade/continual.go

```diff
@@ -67,6 +67,22 @@ func ChannelContinualTest() pkgupgrade.BackgroundOperation {
 		"ChannelContinualTest", setup, verify)
 }
 
+// SourceContinualTest tests source operation in continual manner during the
+// whole upgrade and downgrade process asserting that all event are propagated
+// well.
+func SourceContinualTest() pkgupgrade.BackgroundOperation {
+	setup := func(c pkgupgrade.Context) {
+		// TODO: not yet implemented
```

Comment:
I suppose this comment is redundant? Same for the comment a few lines below

Reply:
I'd leave it be. The `// TODO:` is reported by IDE and other dev tools. Warn log is useful on logs. That's why it's repeated.